gradle/intTestImage.gradle

```diff
@@ -28,20 +29,24 @@ if (useIncomingDistributions) {
     }
 
     // Compensate for the top level dir in the zip
-    def unpackedPath = "$unpackIncomingAllDistribution.destinationDir/${incomingAllDistribution.name - "-all.zip"}"
+    def unpackedPath = "$unpackIncomingAllDistribution.destinationDir/gradle-${incomingDistributionsBuildReceipt.versionNumber}"
 
     intTestImage {
         dependsOn unpackIncomingAllDistribution
-        from unpackedPath
+        from(unpackedPath) {
```

Comment:
let's add some comments describing what we're doing here (switching the "real" version out for our snapshot test version) and why